core/src/main/scala/com/rallyhealth/vapors/factfilter/dsl/ExprBuilder.scala

```diff
@@ -37,10 +37,11 @@ object ExprBuilder extends ExprBuilderSyntax {
 
 trait ExprBuilderLowPriorityImplicits {
 
+  // TODO: Add unit test for this
   implicit def liftFoldableExpr[F[_] : Foldable, V, M[_] : Foldable, U, P](
```

Comment:
I don't think this is used, and I'm not sure it even works. I should write a unit test to verify or remove this code.